### PR TITLE
QT4 create project API tests

### PR DESCRIPTION
**Pull request by Balexer**

### CODE REVIEW

File: Qase_Test/Src/Resources/appSettings.json [hunk 1]
```diff
@@ -3,5 +3,7 @@
   "Browser": "chrome",
   "Timeout": "10",
   "Email": "balexer@mail.ru",
-  "Password": "balexerPassword123"
+  "Password": "balexerPassword123",
+  "Token": "34b9c83661ef70c9d77a0be959aa87781525be9a",
+  "MemberToken": "900bb142759956dc1bf543c10a84c14063cb0361"
```
Comment (ezhukovskaya) [inline Qase_Test/Src/Resources/appSettings.json:8]: please separate broswer settings and user settings

File: Qase_Test/Src/Steps/ApiSteps/Base/BaseApiStep.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        [AllureStep("Authorization")]
+        protected static IRestRequest BaseRequest(Method method, string token, string parameters)
+        {
+            return AddHeaders(method, token)
+                .AddParameter(ContentTypes.ApplicationJson, parameters, ParameterType.RequestBody);
+        }
+
+        [AllureStep("Authorization")]
+        protected static IRestRequest BaseRequest(Method method, string token)
+        {
```
Comment (ezhukovskaya) [inline Qase_Test/Src/Steps/ApiSteps/Base/BaseApiStep.cs]: use lambda expression if there's the only line of code

File: Qase_Test/Src/Steps/ApiSteps/ProjectApiSteps.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+using System.Net;
+using NUnit.Allure.Attributes;
+using Qase_Test.Models;
+using Qase_Test.Steps.ApiSteps.Base;
+using RestSharp;
+
+namespace Qase_Test.Steps.ApiSteps
+{
+    public class ProjectApiSteps : BaseApiStep
```
Comment (ezhukovskaya) [inline Qase_Test/Src/Steps/ApiSteps/ProjectApiSteps.cs]: better to call it ProjectApiHelper, it's a generally accepted decision. We should talk about it personally

File: Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    public class ProjectApiTests
+    {
+        private Project _project;
+        private string _token;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _project = TestDataGeneratorService.GetFakeProject();
+            _token = UserSettings.Token;
```
Comment (ezhukovskaya) [inline Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs]: why can't we keep the token in client, it's always the same, but the tests in the class are reading the settings file every single run in the suite

File: Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+            ProjectApiSteps.CreateProject(_project, _token);
+
+            ProjectApiSteps.DeleteProject(_project, _token).Should().Be(HttpStatusCode.OK);
+        }
+
+        [Test, Description("Create project by member")]
+        [AllureSubSuite("ProjectApi")]
+        public void CreateProjectByMemberTest()
+        {
+            ProjectApiSteps.CreateProject(_project, UserSettings.MemberToken).Should().Be(HttpStatusCode.Forbidden);
```
Comment (ezhukovskaya) [inline Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs]: UserSettings.MemberToken according to the previous comment, you should create default parameter for such cases, but  generally you use only _token

File: Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+            ProjectApiSteps.CreateProject(project, _token).Should().Be(HttpStatusCode.UnprocessableEntity);
+        }
+
+        [Test, Description("Creation project with existing code")]
+        [AllureSubSuite("ProjectApi")]
+        public void CreateProjectWithExistingCodeTest()
+        {
+            ProjectApiSteps.CreateProject(_project, _token).Should().Be(HttpStatusCode.OK);
+
```
Comment (ezhukovskaya) [inline Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs]: who lives in the empty space?

**Review by ezhukovskaya — CHANGES_REQUESTED** [review]

File: Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs (hunk 6 above)
Comment (Balexer) [reply Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs]: Santa Claus, this is his place, he hides there until the new year

**Review by ekaterina-zhukovskaya-simpligov — APPROVED** [review]

**Review by ezhukovskaya — APPROVED** [review]

**Pull request merged**